packages/relay/src/lib/eth.ts

```diff
@@ -1633,6 +1646,47 @@ export class EthImpl implements Eth {
     }
   }
 
+  /**
```

Comment:
Nit. Do we want to document why we are doing this?  I am asking because the `maxAttempts` and `backOff` values are hard coded.  I believe this is a workaround until we get the requested change from the SDK team.

Reply:
What do you mean by document them @ebadiere

Reply:
Is this not a workaround until we get requested changes from the SDK team?  Perhaps a comment stating that might be appropriate, because the `backOff`, `maxAttempts` and `retry`  values are hard coded.

Reply:
Added a comment :)